tracer/src/Datadog.Trace/Ci/Agent/MessagePack/TestCoverageMessagePackFormatter.cs

```diff
@@ -0,0 +1,110 @@
+// <copyright file="TestCoverageMessagePackFormatter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Ci.Coverage.Models.Tests;
+using Datadog.Trace.Vendors.MessagePack;
+
+namespace Datadog.Trace.Ci.Agent.MessagePack;
+
+internal class TestCoverageMessagePackFormatter : EventMessagePackFormatter<TestCoverage>
+{
+    private readonly byte[] _testSessionIdBytes = StringEncoding.UTF8.GetBytes("test_session_id");
```

Comment:
I suddenly had a thought, This is a textbook case for C#11 UTF-8 string literals! 🎉 

```suggestion
    private readonly ReadOnlySpan<byte> _testSessionIdBytes = "test_session_id"u8;
```

And then it hit me - the feature uses `ReadOnlySpan<byte>` _not_ `byte[]` array, so we can't use it because of .NET Framework 😭

Reply:
:(